vspace/src/main/java/edu/asu/diging/vspace/core/exception/ExhibitionDownloadNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package edu.asu.diging.vspace.core.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Exhibition Download folder Not Found")
```

Comment:
incorrect capitalization, should be "Exhibition download folder not found". Also, is the Exhibition download not found or the folder?

And also, let's loose the empty line after the annotation (line 7)